ajax/libs/tablesaw/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "tablesaw",
+  "filename": "tablesaw.min.js",
+  "description": "A set of plugins for responsive tables.",
+  "version": "3.0.9",
+  "author": {
+    "name": "Zach Leatherman",
+    "email": "zach@filamentgroup.com"
+  },
+  "keywords": [
+    "html",
+    "responsive",
+    "rwd",
+    "table",
+    "datatables"
+  ],
+  "homepage": "https://filamentgroup.github.io/tablesaw/demo/kitchensink.html",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/filamentgroup/tablesaw.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/filamentgroup/tablesaw.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(*.scss|*.txt|*.html)",
```

Comment:
Let's use `!(dependencies){,**/!(*.scss|*.txt|*.html)}` to minimize number of the rules.